codegen/templates/python.py.erb

```diff
@@ -0,0 +1,68 @@
+# This code was generated using the code generator from cucumber-messages.
```

Comment:
Whilst it's not mandatory usually we split the two templates up into an enum template and a non-enum (Standard class), template.

Usually this is because the enums generate just some simple readers for each constant whereas the class usually has some primitive behaviour (like serialization / deserialization).

An example of the complexity difference in ruby would be these two

ENUM: Just has constant definitions https://github.com/cucumber/messages/blob/main/ruby/lib/cucumber/messages/attachment_content_encoding.rb

Class: Has some basic readers/serialization logic (Note the attr_readers and the .from_h method https://github.com/cucumber/messages/blob/main/ruby/lib/cucumber/messages/attachment.rb

Reply:
The template is really simple, and all serialization/deserialization is done externally